src/revChatGPT/ChatGPT.py

```diff
@@ -164,7 +164,11 @@ def detect_user_agent(message):
         options.add_argument("--no-sandbox")
         options.add_argument("--disable-setuid-sandbox")
         options.add_argument("--disable-dev-shm-usage")
-        driver = uc.Chrome(enable_cdp_events=True, options=options)
+        driver = uc.Chrome(
+            enable_cdp_events=True, options=options,
+            driver_executable_path=self.config.get("driver_exec_path"),
+            browser_executable_path=self.config.get("browser_exec_path")
```

Comment:
Please make this optional

Reply:
They are optional, if there is no configuration option then it will return `None` which is the default value